backend/inspirehep/records/receivers.py

```diff
@@ -43,7 +43,7 @@ def index_after_commit(sender, changes):
                     change != "delete"
                     and current_app.config["FEATURE_FLAG_ENABLE_AUTHOR_DISAMBIGUATION"]
                 ):
-                    author_disambiguation(model_instance)
+                    disambiguate_authors.delay(str(model_instance.id))
```

Comment:
don't we need the version as well?

Reply:
I checked and we never used this parameter which is [here](https://github.com/inspirehep/inspirehep/blob/a31aae6d3664be8071e2dcf6dd26ff7e49d6246e/backend/inspirehep/disambiguation/tasks.py#L222)
Also, we retry on stale data so I think that it should be ok